### PR TITLE
Changed Notify Function

### PR DESCRIPTION
**Pull request by MarkSkith**
Added a timer parameter to the function notify to allow for autoclosing notification prompts, I also added an if statement to check if callbacks were created in order to determine whether or not to show the accept and decline buttons.

### CODE REVIEW

File: source.lua [hunk 1]
```diff
@@ -499,7 +499,7 @@ do
 	
 	-- new modules
 	
-	function library:Notify(title, text, callback)
+	function library:Notify(title, text, timer, callback)
```
Comment (GreenDeno) [inline source.lua]: The main issue I see here is that you are changing a parameter, not backward compatible. Add the timer parameter after callback.

File: source.lua [hunk 2]
```diff
@@ -657,6 +657,16 @@ do
 			
 			close()
 		end)
+		
+		if not callback then
+			notification.Decline.Visible = false
+			notification.Accept.Visible = false
+			end
+		
+		if timer > 0 then
```
Comment (GreenDeno) [inline source.lua]: Timer should be optional, so you should check if it exists.

File: source.lua [hunk 3]
```diff
@@ -657,6 +657,16 @@ do
...
+			wait(timer)
```
Comment (GreenDeno) [inline source.lua]: The yield should be on another thread as it should not halt the current thread's execution.

**Review by GreenDeno — CHANGES_REQUESTED** [review]

**Comment by MarkSkith** [comment]
Fixed

**Pull request closed without merging**